src/Components/MyProfile.jsx

```diff
@@ -12,22 +180,38 @@ class MyProfile extends Component {
               My profile
             </Header>
             <br></br>
-            <Container className="align-center">
-              <p >
-              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
-              </p>
-              <Divider hidden />
+            <Grid className="align-center">
+              <Grid.Column>
               <p>
                 Please help us make Being Stockholm better and answer six questions about yourself.
-              </p>         
+              </p>
               <Button
                 className='submit-button'
                 href='https://urbanbeings.us18.list-manage.com/subscribe?u=511ba4646c76ccebddfc09524&id=4b6589bfcd'
                 target='_blank'
               >
                 Take me to the survey
               </Button>
-            </Container>
+              </Grid.Column>
+            </Grid>
+
+            <Grid >
+              <Grid.Column id='pending-entries' mobile={16} tablet={16} computer={16} largeScreen={5} widescreen={5}>
```

Comment:
I assume these screen settings are not possible (or easy) to set in site.overrides instead - for consistency?

Reply:
not now man

Reply:
ok